### PR TITLE
Enhancement/scott pd

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: match/scott_pd.py [hunk 1]
```diff
@@ -14,24 +11,26 @@ def prepare_post_data():
...
+    decision = 1
     matcher.save_pairs_to_excel(data_file_path(
         'match/scott_pd_cprr_2020_v_scott_pd_pprr_2021.xlsx'), decision)
     matches = matcher.get_index_pairs_within_thresholds(decision)
-
     match_dict = dict(matches)
+
     cprr.loc[:, 'uid'] = cprr['uid'].map(lambda v: match_dict.get(v, v))
-    return cprr
+    return cprr;pprr
```
Comment (pckhoi) [inline match/scott_pd.py:33]: here only the cprr is changed so you don't need to also return pprr.

**Comment by pckhoi** [comment]
@ayyubibrahimi I don't see you add any fuse script. Is this ready to be reviewed?

**Comment by ayyubibrahimi** [comment]
My bad. It is now ready for review.

File: match/scott_pd.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def match_pprr_and_post(pprr, post):
     dfb.loc[:, 'fc'] = dfb.first_name.map(lambda x: x[:1])
     dfb = dfb.drop_duplicates(subset=['uid']).set_index('uid')
 
-    matcher = ThresholdMatcher(ColumnsIndex(["fc"]), {
+    matcher = ThresholdMatcher(ColumnsIndex([fc'']), {
```
Comment (pckhoi) [inline match/scott_pd.py:45]: This doesn't look like correct syntax. Did you run this script?

Comment (ayyubibrahimi) [reply match/scott_pd.py]: Corrected.

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**